src/Compression/LZ4_decompress_faster.cpp

```diff
@@ -24,6 +24,14 @@
 #include <arm_neon.h>
 #endif
 
+#if defined(__BYTE_ORDER__) && (__BYTE_ORDER__ == __ORDER_BIG_ENDIAN__)
+static inline UInt16 LZ4_readLE16(const void* memPtr)
+{
+        const UInt8* p = reinterpret_cast<const UInt8*>(memPtr);
+        return static_cast<UInt16>(p[0]) + (p[1]<<8);
```

Comment:
Wrong style - should be `p[1] << 8` instead of `p[1]<<8`.

Reply:
Fixed